src/repos.js

```diff
@@ -1,10 +1,13 @@
 const api = require('./api');
 
+/* blacklist of repositories to not include */
+const blacklist = ['rfcs', 'probot-app-workflow'];
```

Comment:
Ideally we pull this out of the util function and let the consumer decide whether they want to apply a blacklist of not... but for now, I think all consumers want to ignore these two repos so this should work.